From ef9b69f9f7a39da95e6d0c8d21c09892d35362ea Mon Sep 17 00:00:00 2001
From: zjp-tju <928048779@qq.com>
Date: Thu, 11 Aug 2022 19:49:16 +0800
Subject: [PATCH] patch2

---
 patch.c | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 patch.c

diff --git a/patch.c b/patch.c
new file mode 100644
index 0000000..68094f3
--- /dev/null
+++ b/patch.c
@@ -0,0 +1 @@
+""sdasdsa
\ No newline at end of file
-- 
2.37.1

